hooks/dot-bash.sh

```diff
@@ -11,8 +11,8 @@ for file in "$@"; do
 
 	# Confirm expected schellcheck header
 	#
-	LINE1="$(head -n 1 "${file}")"
-	if [[ "${LINE1}" != "# shellcheck shell=bash" ]]; then
+	HEADER="$(head -n 2 "${file}")"
```

Comment:
I'm using two lines here, which should be okay.

But maybe we don't even want to check all bash files listed in `clean_files.txt`? It seems to me this was intended to be used only for included files.

Reply:
.bash files are always included in Bash-it, and not invoked directly. `theme.base.bash` is no exception. You should remove the change in the script and remove the shebang from the file as well